Cargo.toml

```diff
@@ -1,7 +1,12 @@
 [workspace]
 resolver = "2"
 members = ["crates/*"]
-default-members = ["crates/symbolicator", "crates/symbolicli", "crates/symsorter", "crates/wasm-split"]
+default-members = [
+    "crates/symbolicator",
+    "crates/symbolicli",
+    "crates/symsorter",
+    "crates/wasm-split",
```

Comment:
Why don't we have other crates in this list?

Reply:
these listed here are the ones that we `cargo build` and ship. all others are just internal.
We run CI and `cargo test` with `--workspace` so everything is tested.